draft-ietf-anima-brski-prm.md

```diff
@@ -336,84 +320,84 @@ Solution examples based on existing technology are provided with the focus on ex
 # Architectural Overview {#architecture}
 
 For BRSKI with pledge in responder mode, the base system architecture defined in BRSKI {{RFC8995}} is enhanced to facilitate the new use cases in which the pledge acts as server.
-The pledge-responder-mode allows delegated bootstrapping using a registrar-agent instead of a direct connection between the pledge and the domain registrar.
+The responder mode allows delegated bootstrapping using a registrar-agent instead of a direct connection between the pledge and the domain registrar.
 
 Necessary enhancements to support authenticated self-contained objects for certificate enrollment are kept at a minimum to enable reuse of already defined architecture elements and interactions.
 
 For the authenticated self-contained objects used for the certification request, BRSKI-PRM relies on the defined message wrapping mechanisms of the enrollment protocols stated in {{req-sol}} above.
 
-The security used within the document for bootstrapping objects produced or consumed by the pledge bases on JOSE {{RFC7515}}. In constrained environments it may be provided based on COSE {{RFC9052}} and {{RFC9053}}.
+The security used within the document for bootstrapping objects produced or consumed by the pledge bases on JOSE {{RFC7515}}.
+In constrained environments it may be provided based on COSE {{RFC9052}} and {{RFC9053}}.
 
 An abstract overview of the BRSKI-PRM protocol can be found in  {{BRSKI-PRM-abstract}}.
 
 
-## Pledge-Responder-Mode (PRM): Registrar-Agent Communication with Pledges {#uc2}
+## Registrar-Agent Communication with Pledges {#uc2}
 
-To support mutual trust establishment between the domain registrar and pledges not directly connected to the customer site/domain, this document specifies the exchange of authenticated self-contained objects (the voucher request/response as known from BRSKI and the enrollment request/response as introduced by BRSKI-PRM) with the help of a registrar-agent.
+To support mutual trust establishment between the domain registrar and pledges not directly connected to the customer site/domain, this document specifies the exchange of authenticated self-contained objects (the voucher-request/response as known from BRSKI and the enrollment-request/response as introduced by BRSKI-PRM) with the help of a registrar-agent.
 
 This leads to extensions of the logical components in the BRSKI architecture as shown in {{uc2figure}}.
 Note that the Join Proxy is neglected in the figure as not needed by the registrar-agent.
```

Comment:
Since the Join Proxy is not discussed in this document, just recurrently that it is omitted, might it not become much easier to say a registrar-proxy MUST be able to connect to the domain registrar and drop the Join Proxy altogether for BRSKI-PRM?

Reply:
We did not want to get rid of the join proxy as it is used in BRSKI. The intention was rather to state that BRSKI-PRM does not hinder its use but also does not require it.  Will change the note to:

Note that the Join Proxy is neglected in the figure. It may be used as specified in BRSKI (RFC 8995) by the  registrar-agent to connect to the registrar.

Reply:
Is a "MAY" then.

Reply:
Okay. True. Changed.